blog/PyMC_Past_Present_Future.md

```diff
@@ -0,0 +1,166 @@
+(PyMC_past_present_future)=
+# PyMC: Past, Present, and Future (PyMCon 2020)
+
+:::{post} January 25, 2023
+:tags: events, community, talks
+:category: news, testimonial
+:author: Sangam SwadiK, Rowan Schaefer
+:image: 0
+:exclude:
+
+Summary of Chris Fonnesbeck's PyMCon 2020 keynote.
+:::
+
+
+- Table of Contents
+    1. Background
+    2. PyMC Development Timeline
+        - Early 2000s
+        - PyMC 1.0
+        - PyMC 2.0
+        - PyMC 3.0
+        - Post-3.0
+    3. Applications
+    4. Present Day
+    5. Contributing
+
+At the [2020 PyMCon conference](https://pymc-devs.github.io/pymcon/about), Chris Fonnesbeck discussed the history and future of PyMC in his talk ["PyMC: Past, Present, and Future"](https://www.youtube.com/watch?v=RXODDVK7B-g&list=PLRD4U7ajWjSORb7M-VjtnvRUrfG6jXqOP&index=4). 
+
+In the talk, he discussed the broader context of probabilistic programming in the early 2000s, outlined the challenges and successes of early development, and provided insights into the future direction of the project. This blog provides a summary of the talk, which can be [viewed in full on Youtube](https://www.youtube.com/watch?v=RXODDVK7B-g&list=PLRD4U7ajWjSORb7M-VjtnvRUrfG6jXqOP&index=4), and a timeline of PyMC’s history up to 2020.
+
+## Background
+
+![Background](../_static/chris-pymcon-keynote/Background.png)
+
+### **What is PyMC?**
+
+[PyMC](https://www.pymc.io/welcome.html) is a powerful and widely used probabilistic programming framework that allows users to implement state-of-the-art Bayesian inference methods in Python. The project was started by Chris Fonnesbeck in 2003 as a graduate student at the University of Georgia, and has since grown to have over 300 contributors.
+
+### **What’s a probabilistic programming language?**
+
+A probabilistic programming language is a language that employs [stochastic](https://en.wikipedia.org/wiki/Stochastic) [primitives](https://en.wikipedia.org/wiki/Primitive_data_type). Just as we have integers and strings and floating point numbers in most languages, a probabilistic programming language will have **random variables** or **probability distributions.**
+
+Why? These stochastic primitives are used as building blocks to build bayesian models. They give us the ability to specify probability models at a very high level. 
+
+By abstracting away much of the underlying machinery that goes into random number sampling and other forms of inference, probabilistic programming makes bayesian inference more accessible to those who are not software developers or statisticians.
+
+# Summary Timeline
+
+![Summary Timeline](../_static/chris-pymcon-keynote/Timeline.png)
+
```

Comment:
@SangamSwadiK 
The timeline is really great! I love the visualization, and I think it's really useful for the reader.
One suggestion I have:
- Ensure that the duration is up to scale.

For example, the block for "1997" is the same width as the one for "2003-2005" and "2006-2013".  Yet, the years covered vary significantly:  
- 1997: 1 year
- 2003-5: 2 years
- 2006-13:  7 years

Reply:
The scale is still off.

1997:  1 unit
2006-2013:  1.5 units  (? but it's 7 years)

<img width="531" alt="Screen Shot 2023-01-29 at 8 20 30 AM" src="https://user-images.githubusercontent.com/2507232/215329059-670c3c66-4e21-4acb-a993-37f5bbd85645.png">

The font is also a bit small to read.

Maybe it needs to be on multiple lines?

Did you do this in Canva? Is there a link?